src/main/java/DBot.java

```diff
@@ -0,0 +1,132 @@
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Hashtable;
+
+public class DBot {
+    private static boolean isOn;
+    private static List<Task> taskList;
+
+    public static void main(String[] args) {
+        isOn = true;
+        taskList = new ArrayList<>();
+
+        final String BREAK_LINE = "____________________________________________________________";
+        final String GREETING_LINE = BREAK_LINE + "\nHello! I'm DBot\nWhat can I do for you?\n" + BREAK_LINE;
+        System.out.println(GREETING_LINE);
+
+        Scanner in = new Scanner(System.in);
+
+        while (isOn) {
+            System.out.print("Command: ");
+            String line = in.nextLine().strip();
+            System.out.println(BREAK_LINE);
+
+            if (line.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                isOn = false;
+            } else if (line.equals("list")) {
+                list();
+            } else if (line.startsWith("mark ")) {
+                mark(line);
+            } else if (line.startsWith("unmark ")) {
+                unmark(line);
+            } else if (line.startsWith("todo ")) {
+                todo(line);
+            } else if (line.startsWith("event ")) {
+                event(line);
+            } else if (line.startsWith("deadline ")) {
+                deadline(line);
+            } else {
+                System.out.println("Unknown command: " + line);
+            }
```

Comment:
Consider using switch case for this case, as the different cases of input is pre-determined